### DISCUSSION TITLE
Multiple assets for same OS under release section?

### DISCUSSION THREAD

**Discussion by mh-saeed** [dbgate/dbgate#882] [General]
there are multiple installers for same OS, which makes me confuse which one to choose.
like there are multiple dmgs files for macOS, so which one should I pick?
**My Machine:** MacBook Pro M2 arm64

**Comment by janproch** [comment]
for community edition: https://github.com/dbgate/dbgate/releases/latest/download/dbgate-latest.dmg
for premium edition: https://github.com/dbgate/dbgate/releases/latest/download/dbgate-premium-latest.dmg

you are right that downloads page needs to be a bit redesigned
